src/pages/auth/reset-password.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import { useNavigate, useParams } from "react-router-dom";
+import { useAuth } from "../../data/authentication";
+import { createToastify } from "../../helpers/toastify";
+
+export const ResetPassword = () => {
+  const params = useParams();
+  const [password, setPassword] = useState({ new: "", confirm: "" });
```

Comment:
Let's better use queryParams for this, to avoid braking the semantic of the url, something/otherSomething means that is accesing a resources from something